### PR TITLE
Add Activity Form

### PR DESCRIPTION
**Pull request by milodevhh**
- Form component
- Handle the submit event
- create a state for activities
- Pass handleAddActivity to the Form component

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,13 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <header></header>
```
Comment (mbosselmann) [inline src/App.js]: If inside the header element is nothing rendered, then you do not need to add one.

File: src/Components/Form/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import Button from "../Button";
+
+export default function Form() {
+  return (
+    <form>
+      <h2>Add new Activity</h2>
+      <div>
```
Comment (mbosselmann) [inline src/Components/Form/index.js:22]: Do you really need this `div` 🤔

**Review by mbosselmann — COMMENTED** [review]
Looks good! :octocat:  But the functionality for adding an activity is missing. :) And I suggest to only add CSS files if you want to add some stylings.

File: src/App.js (hunk 1 above)
Comment (milodevhh) [reply src/App.js]: sure, I added a Header ;)

File: src/Components/Form/index.js (hunk 2 above)
Comment (milodevhh) [reply src/Components/Form/index.js:22]: got it... thx

**Pull request merged**